docs/slides/day2.md

```diff
@@ -0,0 +1,577 @@
+y = -2
+z = 4
+
+if x = 1 || y = -1
+    puts "x = 1 or y = -1"
+elsif z = 4
+    puts "z = 4"
+end
+```
+Reference: [boolean expressions](https://github.com/Ada-Developers-Academy/jump-start-live/blob/master/lessons/day2/boolean_expressions.md)
+
+???
+
+Tricky!
+* A single equals (`=`) is an assignment expression
+* This should be a double equals sign (`==`), an equality check, instead
+* The assignment expression `x = 1` evaluates to `1`, which is a truthy value
+  in Ruby
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+exercise.rb:7: warning: found `= literal' in conditional, should be ==
+x = 1 or y = -1
+```
+
+---
+
+# What will get printed on the terminal?
+
+```ruby
+x = 2
+y = -2
+z = 4
+
+if x == 3 || 4
+    puts "x is 3 or 4"
+else
+    puts "x is not 3 or 4"
+end
+```
+Reference: [boolean expressions](https://github.com/Ada-Developers-Academy/jump-start-live/blob/master/lessons/day2/boolean_expressions.md)
+
+???
+
+More tricks!
+* `4` is a truthy value, so `x == 3 || 4` will always evaluate to true
+* In Ruby, everything is `truthy` except for `nil` and `false`
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+x is 3 or 4
+```
+
+---
+
+# Exercise
+* If the value of the variable `x` is `2`, the value of the variable `y` is `-2`,
+  and the value of the variable `z` is `4`, write a conditional statement that
+  prints `"it's true"` if `x` is greater than `0` and `y` is less than `0`.
+
+* **Challenge** - come up with a different way to write the conditional
+  statement that behaves the same. Hint: use negation (`!`).
+
+```ruby
+x = 2
+y = -2
+z = 4
+```
+
+--
+
+```ruby
+# Option 1
+if x > 0 && y < 0
+    puts "it's true!"
+end
+```
+
+--
+
+```ruby
+# Option 2
+if !(x <= 0 || y >= 0)
+    puts "it's true!"
+end
+```
+
+[De Morgan's laws](https://github.com/Ada-Developers-Academy/jump-start-live/blob/master/lessons/day2/boolean_expressions.md#demorgans-laws)
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+else order == "GRANDE"
+    puts "16"
+end
+```
+
+???
+
+* Everything besides short and tall fall in the `else` block
+* `order == "GRANDE"` is actually a statement within the else, but it was
+  mistakenly put on the wrong line
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): short
+8
+```
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): grande
+16
+```
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): no thank you
+16
+```
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+*else order == "GRANDE"
```

Comment:
Is this * a typo?

Reply:
Good question! It is not - it's the special syntax I mentioned in https://github.com/Ada-Developers-Academy/jump-start-live/pull/31#discussion_r362360895 that allows for highlighting individual lines within a code snippet